yt_dlp/extractor/mtv.py

```diff
@@ -313,6 +313,10 @@ def _extract_mgid(self, webpage):
             video_player = self._extract_child_with_type(ab_testing or main_container, 'VideoPlayer')
             mgid = video_player['props']['media']['video']['config']['uri']
 
+        if not mgid:
```

Comment:
other extractors inherting from `MTVServicesInfoExtractor` still work?

Reply:
Of course, this just added another option to extract the correct mgid if the others couldn't extract it.